src/view/app/Indexes/indexes.tsx

```diff
@@ -32,7 +32,7 @@ function rowKeyGetter(row: IndexRow) {
 }
 
 function Indexes({ tableDetails, configuration }: IConfigProps) {
```

Comment:
can be accepted as a fix, but these props are not needed. Can be removed.
Configuration can be replaced in the following way:
`const configuration = getVSCodeConfiguration();`